docs/tools/locust/taskset.md

```diff
@@ -0,0 +1,308 @@
+---
+sidebar_position: 8
+---
+# TaskSet
+
+Tasks are distinct functions, or callables, that tell Locust what to do during load testing. A task
+is, in essence, a load test. `TaskSet` classes are a way to link tasks together and keep the code
+organized. Task sets and functions should all be defined in python files in the `tasks/` directory.
+
+It is possible for a user class to have more than one task set, but it's important to keep in mind
+is that if a user has more than one task set, they will only ever switch between the task sets if
+you remember to have the task set stop at some point. Otherwise, the user will just stay stuck on
+their first task set until the load tests end. Examples will be included below.
+
```

Comment:
It may be useful to note that the tasksets get run in a randomized order and with the same proportion unless we override the weight per user class or task.

Reply:
Ah good point, I'll add a note about that.